std/http/server.ts

```diff
@@ -401,6 +401,28 @@ export async function listenAndServe(
   }
 }
 
+export type HTTPSOptions = Omit<Deno.ListenTLSOptions, "transport">;
+
+export function serveTLS(options: HTTPSOptions): Server {
```

Comment:
Can you add a jsdoc with a little example?

Reply:
Added.